test/test_helper.rb

```diff
@@ -0,0 +1,18 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+require 'rails/test_help'
+
+class ActiveSupport::TestCase
+  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
+  fixtures :all     
+
+  Time.zone = 'Alaska'
+  Time.now
+  Date.today
+begin
+  
+rescue Exception
```

Comment:
__[rubocop]__
* Lint/HandleExceptions: Do not suppress exceptions. [:ref](https://github.com/bbatsov/ruby-style-guide#dont-hide-exceptions)
* Lint/RescueException: Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`? [:ref](https://github.com/bbatsov/ruby-style-guide#no-blind-rescues)